src/python/dxpy/dx_extract_utils/filter_to_payload.py

```diff
@@ -119,18 +115,20 @@ def location_filter(location_list):
     }
 
     for location in location_list:
-        # First, ensure that the geno bins width isn't greater than 250 megabases
         start = int(location["starting_position"])
-        end = int(location["ending_position"])
-        if end - start > 5000000:
-            err_exit(
-                "Error in location {}\nLocation filters may not specify regions larger than 5 megabases".format(
-                    location
-                )
-            )
+        if "ending_position" in location:
+            end = int(location["ending_position"])
+            # Ensure that the geno bins width isn't greater than 5 megabases
+            if end - start > 5000000:
+                err_exit('\n'.join([
+                    "Error in location {}".format(location),
```

Comment:
Suggestion to use f-strings instead of format for better readability. 

```suggestion
                    f"Error in location {location}",
```

Reply:
I agree it's more readable, but all the format strings are in the old format as a legacy from python 2 compatibility requirements so I trying to conform here.